themes/digital.gov/layouts/partials/core/guides/guide-page-head.html

```diff
@@ -1,29 +1,34 @@
-<header class="page-head page-head-{{- .Type -}}">
+<header class="page-head dg-guide__header">
   {{- partial "core/guides/guide-read-time.html" . -}}
 
   {{- if not .IsNode -}}
     {{- if .Params.kicker -}}
-      <p class="kicker">{{- .Params.kicker -}}</p>
+      <p class="dg-guide__header-kicker">{{- .Params.kicker -}}</p>
     {{- else -}}
-      <p class="kicker">
-        <a href="{{- .Parent.Permalink -}}" title="Go to {{- .Parent.Title -}}"
+      <p>
+        <a
+          class="dg-guide__header-kicker"
+          href="{{- .Parent.Permalink -}}"
+          title="Go to {{ .Parent.Title }}"
           >{{- .Parent.Title -}}</a
         >
       </p>
     {{- end -}}
 
     {{/* Page Title */}}
-    <h1>{{- .Title | markdownify -}}</h1>
+    <h1 class="dg-guide__header-title">{{- .Title | markdownify -}}</h1>
 
     {{/* Deck */}}
     {{- if .Params.deck -}}
-      <div class="usa-intro">{{- .Params.deck | markdownify -}}</div>
+      <div class="dg-guide__header-intro">
+        {{- .Params.deck | markdownify -}}
+      </div>
     {{- end -}}
   {{ else }}
     {{- if .Params.kicker -}}
-      <p class="kicker">{{- .Params.kicker -}}</p>
+      <p class="dg-guide__header-kicker">{{- .Params.kicker -}}</p>
     {{- else -}}
-      <p class="kicker">{{- .Page.Title -}}</p>
+      <p class="dg-guide__header-kicker">{{- .Page.Title -}}</p>
```

Comment:
We should refactor this in the future so it's less repetitive.

```
<p class="dg-guide__header-kicker>
  {{- if .Params.kicker -}}
     {{ .Params.kicker }}
  {{  else }}
     {{ .Page.Title }}
  {{ end }}
</p>
```

---

We could take it even further by creating a kicker component, but that's a future ticket.

Reply:
Good idea, card created for this refactor.